srcpkgs/Brainfunk/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'Brainfunk'
+pkgname=Brainfunk
+version=0.9.0
+revision=1
+wrksrc=${pkgname}-${version}-beta
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="png++"
```

Comment:
Suspect you want `makedepends` here.

Reply:
What I just noticed as well, before checking this.  Makes more sense, and it should work.